react/.storybook/preview.js

```diff
@@ -1,58 +1,92 @@
-import { withA11y } from '@storybook/addon-a11y';
-import { withInfo } from '@storybook/addon-info';
+import React from 'react';
+import ReactDOMServer from 'react-dom/server';
+import {
+  Title,
+  Subtitle,
+  Primary,
+  ArgsTable,
+  Heading,
+  ComponentsTable,
+  Stories,
+  PRIMARY_STORY,
+  CURRENT_SELECTION,
+  SourceState,
+  DocsContext,
+  Canvas, Story
+} from '@storybook/addon-docs/blocks';
+import { ActionBar, Source } from '@storybook/components';
+import prettier from 'prettier/standalone';
+import parserHtml from 'prettier/parser-html';
+import { Prism as SyntaxHighlighter } from 'react-syntax-highlighter';
+import { okaidia } from 'react-syntax-highlighter/dist/esm/styles/prism';
 import '../src/index.scss';
 
 const storyKindOrder = [
   'about', // storyKindOrder.indexOf -1 follow alphabetical order
   'brand', // storyKindOrder.indexOf -1 follow alphabetical order
   'dataviz', // storyKindOrder.indexOf -1 follow alphabetical order
-  'forms|atoms',
-  'forms|molecules',
-  'forms|organisms',
-  'forms|context',
+  'forms',
   'atoms',
   'molecules',
   'organisms',
-  'templates',
-  'pages'
+  'others/templates',
+  'others/pages'
 ];
 
-addParameters({
-  options: {
-    storySort: (a, b) => {
-      const aKind = a[1].kind.split('/')[0];
-      const bKind = b[1].kind.split('/')[0];
-      return (storyKindOrder.indexOf(aKind) - storyKindOrder.indexOf(bKind)) || a[1].id.toLowerCase().localeCompare(b[1].id.toLowerCase(), undefined, { numeric: true });
-    }
-  }
-});
-
-addDecorator(
-  withInfo({
-    styles: {
-      button: {
-        base: {
-          fontFamily: 'sans-serif',
-          fontSize: '14px',
-          fontWeight: '500',
-          display: 'block',
-          position: 'fixed',
-          border: 'none',
-          background: '#14558f',
-          color: '#fff',
-          padding: '5px 15px',
-          cursor: 'pointer',
-        },
-        topRight: {
-          bottom: 0,
-          right: 0,
-          top: 'unset',
-          borderRadius: '5px 0 0 0',
-        }
+export const StoryPage = ({ StoryComponent = null, showStories = false, Description }) => {
+  const docsContext = React.useContext(DocsContext);
+  const [showHTML, setShowHTML] = React.useState(false);
+  const { id, name, parameters = {}, args } = docsContext;
+  const { component } = parameters;
+  const HtmlComponent = StoryComponent || component;
+  let html = null;
+  if (HtmlComponent) {
+    html = prettier.format(ReactDOMServer
+    .renderToStaticMarkup(
+      (
+        <HtmlComponent {...args} />
+      )),
+      {
+        htmlWhitespaceSensitivity: 'ignore',
+        endOfLine: 'auto',
+        parser: 'html',
+        plugins: [parserHtml]
       }
-    }
-  })
-);
+    );
+  }
+  const actionItem = {
+    title: showHTML ? 'Hide HTML' : 'Show HTML?', 
+    onClick: () => setShowHTML((prev) => !prev)
+  };
+  return(
+    <>
+      <Title>Description</Title>
```

Comment:
can the title be the component name instead?
```suggestion
      <Title>{StoryComponent.displayName}</Title>
```